src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```
@@ -162,36 +162,81 @@ admin_products_v2_duplicate:
   requirements:
     productId: \d+
 
-admin_products_v2_toggle_status:
-  path: /{productId}/toggle_status/{shopId}
+admin_products_v2_duplicate_shop:
+  path: /{productId}/duplicate/shop/{shopId}
```

https://devdocs.prestashop-project.org/8/development/naming-conventions/#routes-and-paths
`PrestaShop routes follow admin_{resource}_{action} naming structure....
When the route points to an action to be performed on an individual resource then URL path should follow /{resource}/{resourceId}/{action} naming (e.g. /customers/23/edit).`
So the action name shouldn't be separated by `/`, so IMO it should look like this at least:
```suggestion
  path: /{productId}/duplicate-shop/{shopId}
```